### PR TITLE
nested collections in response, no property delete by omission

### PR DESCRIPTION
**Pull request by builtbywill**

### CODE REVIEW

**Comment by builtbywill** [comment]
@tragessere updates pushed

File: src/fetchSaga.ts [hunk 1]
```diff
@@ -10,15 +10,29 @@ import {
 	EndpointMappings,
 	ErrorFunction,
 	FetchAction,
+	FetchConfig,
 	FetchError,
 	LoggerFunction,
 	Metadata,
```
Comment (tragessere) [inline src/fetchSaga.ts]: The only thing I see is this Metadata import not being used anymore.

Comment (builtbywill) [reply src/fetchSaga.ts]: @tragessere removed

File: src/fetchReducer.test.ts [hunk 2]
```diff
@@ -94,62 +95,121 @@ describe('supporting functions', () => {
...
 				child: {
 					id: 1,
 					foo: false
 				}
 			}
 			const incoming = { id: 1, name: 'Bob' }
-			expect(mergeRelations(current, incoming)).toEqual({
+			expect(merge(current, incoming)).toEqual({
+				id: 1,
+				name: 'Bob',
```
Comment (jkdufair) [inline src/fetchReducer.test.ts]: Awesome!

**Comment by camliu89** [comment]
:thumps:

**Pull request merged**